modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageManagerImpl.java

```diff
@@ -319,6 +321,19 @@ public CompletableFuture<Entry> get(ByteArray key, long revUpperBound) {
         }
     }
 
+    @Override
+    public List<Entry> getEntriesLocally(byte[] key, long revLowerBound, long revUpperBound) {
```

Comment:
Let's be consistent with other methods naming. There are bunch of method in `MetaStorageManager`, `RocksDbKeyValueStorage` and other related places that return entries, however neither of them have entries in it's name, you on the contrary introduces getEntriesLocally. Please rename it and all related `getEntries`, `doGetEntries`, etc.